### DISCUSSION TITLE
How to Get the TUI mode on AWS EKS installation using Helm

### DISCUSSION THREAD

**Discussion by sriniazhagar** [JanssenProject/jans#5990] [Q&A]
Hi Team, 

I was successfully done the installation and did the health check up. So my services are running properly.

But i couldn't able to get the TUI mode on AWS EKS platform installation.

Can someone please help how to proceed further.

**Comment by moabu** [comment]
Hey @sriniazhagar ,


Here is an answer for your question https://github.com/JanssenProject/jans/discussions/5692#discussioncomment-6591282

**Comment by ossdhaval** [comment]
@sriniazhagar We are adding documentation that addresses your question. Being tracked using #5996.

**Reply by ossdhaval** [reply]
Hi @sriniazhagar PR above has been merged (thanks to @misba7 and @moabu for video). The document is available [here](https://docs.jans.io/head/admin/kubernetes-ops/tui-k8s/)

**Reply by sriniazhagar** [reply]
Thank you so much for all your support and response.

**Comment by sriniazhagar** [comment]
Required information is shared, so closing this ticket.
